server/config/route.js

```diff
@@ -32,6 +33,9 @@ module.exports = function(app, express) {
   app.post('/api/comment/:user_id', postController.addComment);
   app.post('/api/signin/', userController.addUser);
 
+  app.get('/api/guides/', guideController.getAll);
```

Comment:
I did that to match the format of the existing routes (technically, it doesn't matter which way we do it, but it's sort of a convention having to do with single resources vs collections). 

See first answer here: http://programmers.stackexchange.com/questions/186959/trailing-slash-in-restful-api
Or this article: https://cdivilly.wordpress.com/2014/03/11/why-trailing-slashes-on-uris-are-important/